lib/ability.rb

```diff
@@ -70,6 +69,8 @@ def initialize(user, resource, controller)
           UserPriceGroupMember,
         ]
 
+        can :read, Notification
```

Comment:
This actually changes the behavior of Notifications somewhat. I could probably talked in to the new behavior, but there are a couple inconsistencies.

It used to be if you had Notifications, then they were always visible even when you weren't in an admin context, because they're scoped to the user. Now we're only showing the Notifications while you're in a facility admin context, however we're still only scoping the notifications to the user, so if the user has admin rights on multiple facilities, and they have notifications on one facility, they'll still appear while in the context of another facility.